skimage/color/tests/test_colorlabel.py

```diff
@@ -38,6 +38,17 @@ def test_wrong_kind():
         label2rgb(label, kind='foo', bg_label=-1)
 
 
+def test_uint_image():
```

Comment:
this test is failing on master, img was not rescaled with `img_as_float` and therefore kept a 0-255 range while the color mask to be added was in the [0-1] range so not visible when superimposed. The output was float but in [0-255] so also matplotlib would represent it completely white in the gallery examples.